Create-Cluster.md

```diff
@@ -3,66 +3,90 @@
 ## Prerequisites
 
 ```
-
+$ brew install kops
 $ brew install kubernetes-cli
 $ brew install kubernetes-helm
 $ brew install terraform
+$ git clone git@github.com:ministryofjustice/kubernetes-investigations.git
+$ git clone git@github.com:ministryofjustice/cloud-platform-environments.git
+$ git clone git@github.com:ministryofjustice/laa-fee-calculator.git
 
+Have access to the moj-cloud-platforms-dev Auth0 tenant 
```

Comment:
Perhaps this should say, if you don't, follow the steps below to create a new one.